src/styles/rewiews.scss

```diff
@@ -0,0 +1,16 @@
+.rewiew {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $boldTextColor;
+  width: $defaultWidth;
```

Comment:
The variable `$defaultWidth` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.